firefox-ios/firefox-ios-tests/Tests/ClientTests/ContentBlockerTests.swift

```diff
@@ -21,9 +21,12 @@ final class ContentBlockerTests: XCTestCase {
 
     func testCompileListsNotInStore_callsCompletionHandlerSuccessfully() {
         let expectation = XCTestExpectation()
+        let startDate = Date()
         ContentBlocker.shared.compileListsNotInStore {
+            let finishDate = Date()
+            NSLog("MT Test Time - \(finishDate.timeIntervalSince(startDate))")
```

Comment:
Going to tag @isabelrios and @clarmso in case you do something specific for performance integration testing